diff --git a/node_modules/react-native-vision-camera-face-detector/android/build.gradle b/node_modules/react-native-vision-camera-face-detector/android/build.gradle
index 237b232..fdb1e2a 100644
--- a/node_modules/react-native-vision-camera-face-detector/android/build.gradle
+++ b/node_modules/react-native-vision-camera-face-detector/android/build.gradle
@@ -19,6 +19,7 @@ buildscript {
 }
 
 android {
+    namespace "com.visioncamerafacedetector"
     buildToolsVersion = safeExtGet("VisionCameraFaceDetector_buildToolsVersion", "35.0.0")
     ndkVersion safeExtGet("VisionCameraFaceDetector_ndkVersion", "27.3.13750724")
     defaultConfig {
diff --git a/node_modules/react-native-vision-camera-face-detector/android/src/main/AndroidManifest.xml b/node_modules/react-native-vision-camera-face-detector/android/src/main/AndroidManifest.xml
index d9b6fcc..cfe6513 100644
--- a/node_modules/react-native-vision-camera-face-detector/android/src/main/AndroidManifest.xml
+++ b/node_modules/react-native-vision-camera-face-detector/android/src/main/AndroidManifest.xml
@@ -1,4 +1,4 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.visioncamerafacedetector">
+  >
 
 </manifest>
